source/server-selection/tests/README.rst

```diff
@@ -65,3 +65,37 @@ the TopologyDescription. Each YAML file contains a key for these stages of serve
 Drivers implementing server selection MUST test that their implementation
 correctly returns the set of servers in ``in_latency_window``. Drivers SHOULD also test
 against ``suitable_servers`` if possible.
+
+Selection Within Latency Window Tests
```

Comment:
Now that the "algorithm" is much simpler, is it still worth it to have all drivers implement these unit tests?

Reply:
We're limited a bit by the topology. We could include some of those experiments I did as prose tests--for example, describe a test against a two mongos sharded topology where one of the mongoses has a failpoint that makes every operation take 500ms, then do a ton of concurrent stuff and then assert that the non-failpoint node got picked a lot more. While this does verify the operationCount behavior, I'm not sure whether this is preferable to having a bunch of unit spec tests, though.

Reply:
Please open a jira ticket so we can further discuss adding that prose test. I'm in favor of it.

Reply:
Yeah let's just add it now. I doubt the unified test format is expressive enough for a test like this so it'll need to be a prose test like you said. You can use the blockConnection option for failCommand like this:
```
db.adminCommand({
    configureFailPoint: "failCommand",
    mode: {times: 100000},  # or "alwaysOn"
    data: {
      failCommands: ["find"],
      blockConnection: true,
      blockTimeMS: 500,
      appName: "loadBalancingTest",
    },
});
```

Reply:
done. When you run it, can you let me know what you get for % of operations routed to the slow node? Curious to see if theres any differences across drivers.

Reply:
Implemented. I ran it a few times and see:
```
{('localhost', 27017): 11, ('localhost', 27018): 89}
{('localhost', 27017): 12, ('localhost', 27018): 88}
{('localhost', 27017): 13, ('localhost', 27018): 87}
{('localhost', 27017): 15, ('localhost', 27018): 85}
{('localhost', 27017): 17, ('localhost', 27018): 83}
```

Exciting to see this feature in action!

Reply:
I get around 12-15% too, so it seems like our implementations are consistent, nice!